resources/public/api/conf/1.0/common/errors.yaml

```
@@ -15,6 +15,95 @@ components:
         code: FORMAT_CHECKPOINT_ID
         message: The provided checkpoint ID is invalid
 
+    formatTypeOfBusiness:
+      description: |
+        The format of the type of business field is not valid.
+      value:
+        code: FORMAT_TYPE_OF_BUSINESS
+        message: The provided typeOfBusiness field is invalid
+
+    formatTaxYear:
+      description: The format of the supplied taxYear value is not valid.
+      value:
+        code: FORMAT_TAX_YEAR
+        message: The provided tax year is invalid
+
+    formatBusinessId:
+      description: |
+        The format of the supplied Business ID field is not valid.
+      value:
+        code: FORMAT_BUSINESS_ID
+        message: The provided Business ID is invalid
+
+    formatDate:
+      description: |
+        The supplied date format is not valid.
+      value:
+        code: FORMAT_DATE
+        message: The supplied date format is not valid
+
+    formatAccountingType:
+      description: |
+        The format of the accounting type field is not valid.
+      value:
+        code: FORMAT_ACCOUNTING_TYPE
+        message: The provided accountingType field is invalid
+
+    formatPostcode:
+      description: |
+        The format of the postcode field is not valid.
+      value:
+        code: FORMAT_POSTCODE
+        message: The provided businessAddressPostcode is invalid
+
+    formatLatencyIndicator:
+      description: |
+        The format of a latency indicator fields must be "A" or "Q".
```

@harry-ritchie Can you change these (as well as in the tech spec) to explain what "A" and "Q" mean? Not sure how obvious it is, but since this is supposedly the only API that allows the creation of businesses, vendors might not have a better place to find this out.

e.g.
```suggestion
        The format of a latency indicator fields must be "A" or "Q" (Annual/Quarterly).
```

---

I commented in the wrong place, but my preferences are, in order:
1. in both the schema and error description.
2. only in the schema

---

I'm ok with the changes, as long as the content writers agree. Let me know when they get back to you.

---

Content writer is happy with the changes made to the latency field descriptions in the schema.